sympy/core/relational.py

```diff
@@ -1075,9 +1081,383 @@ def _n2(a, b):
+        if (l.exp.is_integer and Mod(l.exp, 2) in (0, 1)):
+            return l
+    elif l.base.is_extended_nonnegative is None:
+        #assert None,(20.5)
+        return
+
+
+@dispatch(Pow, Rational) # type: ignore
+def _eval_is_ge(_p, r):  # :F811
+    b, e = _p.as_base_exp()
+    if _big_ppow(b, e) and is_ge(S.One, r):
+        return True
+    if _small_ppow(b, e) and is_ge(S.Zero, r):
+        return True
+    if _small_npow(b, e) and is_ge(S.NegativeOne, r):
+        return True
+
+
+@dispatch(Rational, Pow) # type: ignore
+def _eval_is_ge(r, _p):  # noqa:F811
+    b, e = _p.as_base_exp()
+    if r < 1 and r > 0:
+        return is_ge(_invpow(_p), 1/r)
+    if _small_ppow(b, e) and is_ge(r, S.One):
+        return True
+    if _small_npow(b, e) and is_ge(r, S.Zero):
+        return True
+    if _big_npow(b, e) and is_ge(r, S.NegativeOne):
+        return True
+
+
 @dispatch(Expr, Expr)
-def _eval_is_ge(lhs, rhs):
-    return None
+def _eval_is_ge(lhs, rhs):  # noqa:F811
+    from sympy.simplify.radsimp import fraction
+    hit = 0
+    for i in (lhs, rhs):
+        if not i.is_extended_real:
+            ilhs = _invpow(lhs)
+            irhs = _invpow(rhs)
+            if ilhs.is_extended_real and irhs.is_extended_real:
+                return is_ge(irhs, ilhs)
+            return
+        if i.is_Mul or i.is_Pow or i.is_Add:
+            hit += 1
+    if not hit:
+        return
+    ix = False  # keep track of whether L and R were exchanged
+    L, R = lhs, rhs
+    if L.is_Rational:
+        L, R = R, L
+        ix = ~ix
+    if L.is_number and R.is_number:
+        L, R = L - R, S.Zero
+    if not R:
+        if not L:
+            return True  # lhs == rhs already handled before getting here
+        if L.is_extended_negative:
+            return bool(ix)
+        elif L.is_extended_positive:
+            return bool(~ix)
+        elif L.is_extended_nonnegative:
+            return None if ix else True
+        elif L.is_extended_nonpositive:
+            return True if ix else None
+        if L.is_Add:
+            a, b = L.as_two_terms()
+            if a.is_Rational:
+                L = b
+                R = -a
+            else:
+                L = a
+                R = -b
+    # simplify wrt Rationals
+    m, l, b = _aslinear(L)
+    l = _trivial(l)
+    if l is None:
+        #assert None,(11)
+        return
+    mr, r, br = _aslinear(R)
+    r = _trivial(r)
+    if r is None:
+        #assert None,(10)
+        return
+    # make processing canonical
+    if l.is_Number and not r.is_Number:
+        l, r = r, l
+        m, mr = mr, m
+        b, br = br, b
+        ix = ~ix
+    if SYMPY_DEBUG:
+        print(lhs, rhs, '-trivial->', l, r)
+        if pause: input('press return to continue')
+
+    # work really hard to identify a power for comparison
+    if l.is_Pow and r == l.base:
+        r = Pow(r, 1, evaluate=False)
+    elif r.is_Pow and l == r.base:
+        l = Pow(l, 1, evaluate=False)
```

Comment:
The idea behind introducing multiple dispatch here was supposed to be that we wouldn't just put everything in an `Expr,Expr` handler.

Reply:
Are we supposed to create a bunch of permutations of the types:

Add, Add
Add, Pow
Pow, Add
Pow, Pow
Add, Mul
Mul, Add
Pow, Mul
Mul, Pow

Reply:
That was the intention. Maybe it wasn't the best design if that can't be made to work.

This was intended to be more or less a part of the old assumptions and the expectation was not that it would resolve all cases. Instead only "easy" cases would be resolved. The new assumptions are a more natural place to have a slower but more complete implementation for resolving inequalities like this.

Reply:
I'll give it a try to put it into the handlers. The good news is that already tests are running significantly faster with even these changes in place.